### ISSUE TITLE
OpenMM-ML wrapper

### ISSUE THREAD

**Issue by peastman** [JingHuangLab/openmm_deepmd_plugin#21]
Very nice plugin!  Would you be interested in exposing it through [OpenMM-ML](https://github.com/openmm/openmm-ml)?  It provides a standard interface for creating simulations with ML potentials.  You just have to write two Python classes to implement the interface.  See [mlpotential.py](https://github.com/openmm/openmm-ml/blob/main/openmmml/mlpotential.py) for the generic interface and [anipotential.py](https://github.com/openmm/openmm-ml/blob/main/openmmml/models/anipotential.py) for an example of implementing a specific model.  If you're interested I can help.

**Comment by dingye18** [comment]
Yeah, of course. We really enjoyed exposing `DeepmdForce` to OpenMM-ML. 
Before integrating the `DeepmdPlugin` with OpenMM-ML, I need to build the conda packaging script and upload it to conda-forge chanel.
This way we can import `DeepmdForce` in `dppotential.py`. After packaging, we will submit a pull request to OpenMM-ML. 
Any suggestions and comments on this plugin and PR are welcome.

(Sorry for delayed so long to reply this issue)

**Comment by dingye18** [comment]
To do for the integration of `DeepmdPlugin` and `OpenMM-ML`

- [ ] Conda packaging on the OpenMM-Deepmd Plugin
- [ ] Pull request for `dppotential.py`
- [ ] Documentation of the `dppotential.py` on OpenMM-ML.

**Comment by varun-go** [comment]
Is there a timeline for when this wrapper will be created? I am very interested in using it! Thanks

**Comment by dingye18** [comment]
Hope I can finish it within this week. Sorry for delayed so long......

**Comment by dingye18** [comment]
Submit a PR in [#99](https://github.com/openmm/openmm-ml/pull/99)
